matrix.java

```diff
@@ -0,0 +1,38 @@
+/*
+Time Complextiy : O(n+m)
+Space Complexity : O(1)
```

Comment:
Please specify what `m` and `n` indicate.

Reply:
Here, n is number of rows and m is number of columns.